interfaces/builtin/unity7.go

```diff
@@ -194,6 +194,11 @@ dbus send
 owner @{HOME}/.local/share/mime/**   r,
 owner @{HOME}/.config/user-dirs.dirs r,
 
+# gtk settings
```

Comment:
How are we expecting processes to access the real home directory given that HOME is changed?